docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,174 @@
+   create_autofill_manager_handle()
+   {
+       autofill_manager_h amh;
+       int ret;
+       ret = autofill_manager_create(&amh);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+4. When you no longer need the autofill manager library, destroy the autofill manager handle using the `autofill_manager_destroy()` function:
+
+   ```c
+   void
+   destroy_autofill_manager_handle(autofill_manager_h amh)
+   {
+       int ret;
+
+       ret = autofill_manager_destroy(amh);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+   > **Note**
+   >
+   > Do not use the `autofill_manager_destroy()` function in a callback function.
+
+5. After you create the autofill manager handle, connect the background autofill daemon with the `autofill_manager_connect()` function.
+   The function is asynchronous and you can get the result with callback function that registered as the second parameter of `autofill_manager_connect()`.
+
+   ```c
+   static void manager_connection_status_changed_cb(autofill_manager_h amh, autofill_manager_connection_status_e status, void *user_data)
+   {
+       switch (status) {
+           case AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED:
+               dlog_print(DLOG_INFO, LOG_TAG, "connected");
+               break;
+           case AUTOFILL_MANAGER_CONNECTION_STATUS_DISCONNECTED:
+               dlog_print(DLOG_INFO, LOG_TAG, "disconnected");
+               break;
+           case AUTOFILL_MANAGER_CONNECTION_STATUS_REJECTED:
+               dlog_print(DLOG_INFO, LOG_TAG, "rejected");
+               break;
+           default:
+               break;
+   }
+
+   void
+   connect_autofill_daemon(autofill_manager_h amh)
+   {
+       int ret;
+       ret = autofill_manager_connect(amh, manager_connection_status_changed_cb, NULL);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+   You can use the autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
+
+## Getting the installed autofill service list
+
+To get the installed autofill service list:
+
+1. Create the `autofill_service_info_cb()` callback function, which is invoked for each installed autofill service application and is used to access autofill service information:
+
+```c
+#include <app_manager.h>
+
+static bool autofill_service_info_cb(const char* app_id, void* user_data)
+{
+    app_info_h app_h;
+    char *label = NULL;
+
+    app_info_create(app_id, &app_h);
+    app_info_get_label(app_h, &label);
+
+    dlog_print(DLOG_INFO, LOG_TAG, "app id : %s, label : %s", app_id, label);
+
+    if (label)
+        free(label);
+
+    app_info_destroy(app_h);
+
+    return true;
+}
+```
+
+2. Use the `autofill_manager_foreach_autofill_service()` function to get the installed autofill service by invoking a callback function for each installed autofill service application:
```

Comment:
Please change to:

To get the installed Autofill service by invoking a callback function for each installed Autofill service application, use the `autofill_manager_foreach_autofill_service()` function: